students/luke/s07/html_render.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+"""
+A class-based system for rendering html.
+"""
+
+# This is the framework for the base class
+class Element(object):
+
```

Comment:
FYI you only need to add object in python 2x; it's implicit in python 3x

    class Element():